src/Pmi/EntityManager/EntityManager.php

```diff
@@ -51,4 +58,33 @@ public function getTimezone()
     {
         return $this->timezone;
     }
+
+    protected function parseMultipleTimestamps(array $result)
+    {
+        foreach ($result as $key => $value) {
+            $result[$key] = $this->parseTimestamps($value);
+        }
+        return $result;
+    }
+
+    protected function parseTimestamps(array $result)
+    {
+        foreach ($result as $key => $value) {
+            if (null !== $value && substr($key, -3, 3) == '_ts' && preg_match("/^\d{4}\-\d{2}\-\d{2}/", $value)) {
```

Comment:
I can see that this is borrowed from elsewhere (and thereby already vetted for why it does this), but I feel like there are more simplified ways to validate a timestamp?